rules/windows/credential_access_dump_registry_hives.toml

```
@@ -0,0 +1,54 @@
+[metadata]
+creation_date = "2020/11/23"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/23"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempts to export a registry hive which may contain credentials using the Windows reg.exe tool.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+max_signals = 33
+name = "Credential Acquisition via Registry Hive Dumping"
+references = ["https://medium.com/threatpunter/detecting-attempts-to-steal-passwords-from-the-registry-7512674487f8"]
+risk_score = 73
+rule_id = "a7e7bfa3-088e-4f13-b29e-3986e0e756b8"
+severity = "high"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Credential Access"]
+type = "eql"
+
+query = '''
+process where event.type in ("start", "process_started") and
+ process.pe.original_file_name == "reg.exe" and
+ process.args in ("save", "SAVE", "export", "EXPORT") and
+ process.args in ("hklm\\sam",
+                  "hklm\\security",
+                  "HKLM\\SAM",
+                  "HKLM\\SECURITY",
+                  "HKEY_LOCAL_MACHINE\\SAM",
+                  "HKEY_LOCAL_MACHINE\\SECURITY",
+                  "hkey_local_machine\\sam",
+                  "hkey_local_machine\\security"
+                  ) and
+ not wildcard(process.parent.executable, "C:\\Program Files*\\Rapid7\\Insight Agent\\components\\insight_agent\\*\\ir_agent.exe")
```

You can now do [multiple parameters](https://github.com/elastic/elasticsearch/issues/64388) in `:` for case insensitive compares. You can also now [include wildcards](https://github.com/elastic/elasticsearch/issues/65154) in `:` as well.

```suggestion
process where event.type in ("start", "process_started") and
  process.pe.original_file_name : "reg.exe" and
  process.args : ("save", "export", "hklm\\sam", "hklm\\security",
                  "hkey_local_machine\\sam", "hkey_local_machine\\security") and
  not process.parent.executable : "C:\\Program Files*\\Rapid7\\Insight Agent\\components\\insight_agent\\*\\ir_agent.exe"
```

This still needs to be added to py-eql and so will fail unit tests until that is added

(cc: @rw-access)

---

tested on 7.10 its case sensitive. also logic here is "save or export" AND sam (not save or system or security).